examples/confluent_cloud.py

```diff
@@ -50,14 +50,16 @@
 
 from confluent_kafka import Producer, Consumer
 
+def error_cb(err):
+    print(err.str())
 
 p = Producer({
     'bootstrap.servers': '<ccloud bootstrap servers>',
     'sasl.mechanisms': 'PLAIN',
     'security.protocol': 'SASL_SSL',
     'sasl.username': '<ccloud key>',
     'sasl.password': '<ccloud secret>'
-})
```

Comment:
It may look a read a little easier if we set the error cb in the conf dict itself. 

i.e. https://github.com/confluentinc/confluent-kafka-python/blob/master/tests/integration/integration_test.py#L369